specification/resources/spaces/key_get.yml

```
@@ -0,0 +1,40 @@
+operationId: spacesKey_delete
```

```suggestion
operationId: spacesKey_get
```

The operation name must be unique and should reflect the purpose of the endpoint. It is used to generate client method names. This is the cause of the lint error in CI.